python/ray/__init__.py

```diff
@@ -33,6 +33,15 @@
 
 try:
     import pyarrow  # noqa: F401
+
+    # pyarrow is not imported inside of _raylet because of the issue described
+    # above. In order for Cython to compile _raylet, pyarrow is set to None
```

Comment:
`above` -> `below`?

Reply:
No, above is correct. It refers to the comment above `import ray._raylet`. Is this confusing?

Reply:
I see. I thought you were referring this comment `In order for Cython ...`.